requirements.txt

```diff
@@ -1,18 +1,23 @@
+PyQt6
 basemap
 beautifulsoup4
 cartopy
+flask
 gdal < 3.10.0
 geopandas
 matplotlib
 netcdf4
 osmnx
 pandas
 pillow
+pyautogui
 pyproj
-PyQt6
+pywhatkit
 rasterio
 requests
 requests-cache
 scipy
+selenium
 tqdm
+uuid
 xarray
```

Comment:
Need to update the `uv` dependencies and lockfile to reflect the change made to the dependencies.